compiler/include/symbol.h

```diff
@@ -716,6 +716,7 @@ extern const char* astr_coerceCopy;
 extern const char* astr_coerceMove;
 
 void initAstrConsts();
+void initTypeHelperNames();
```

Comment:
You are right, it should go into visibleFunctions.h.
I will move it there. The downside is that driver.cpp will need to #include that header.

Reply:
> The downside is that driver.cpp will need to #include that header.

Yeah, I wondered if that's what this was trying to avoid.  Still, I think it's the right call, thanks!